examples/knockback/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+	"backbone.localStorage": "^1.1.3",
+	"jquery": "^2.0.0",
+	"todomvc-app-css": "^1.0.0",
+	"todomvc-common": "^1.0.1"
```

Comment:
Oh, sorry about that. Fixed: https://github.com/tastejs/todomvc/commit/64484b9e18eabed01b8314af899f50697d167946